xml/System.Threading/Thread.xml

```diff
@@ -1826,8 +1826,8 @@ Main thread: ThreadProc.Join has returned.  Press Enter to end program.
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Gets the cached processor ID used as a hint for which per-core stack to access. It is periodically refreshed to trail the actual thread core affinity.</summary>
```

Comment:
This isn't quite right (the comment in the source is stale).  It should be tweaked to something more like:
"Gets an ID used to indicate on which processor the current thread is executing."
with remarks like:
"This value is cached and periodically refreshed.  It may be stale by the time it's used: consuming code must not rely on its accuracy for correctness."

Reply:
Thanks for the suggestion. I was not aware it was stale. I'll apply it:

```suggestion
        <summary>Gets an ID used to indicate on which processor the current thread is executing.</summary>
```